coverage/Makefile

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright 2014 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
2019

Reply:
I'm planning on eventually integrating these coverage changes with [codecov.io](codecov.io) so that Travis uploads coverage metrics, and the CodeCov bot comments on Pull Requests with coverage deltas [like this.](https://docs.codecov.io/docs/pull-request-comments)